substrate/primitives/staking/src/lib.rs

```diff
@@ -271,7 +272,7 @@ pub trait StakingInterface {
 	/// Checks whether an account `staker` has been exposed in an era.
 	fn is_exposed_in_era(who: &Self::AccountId, era: &EraIndex) -> bool;
 
-	/// Return the status of the given staker, `None` if not staked at all.
+	/// Return the status of the given staker, `Err` if not staked at all.
 	fn status(who: &Self::AccountId) -> Result<StakerStatus<Self::AccountId>, DispatchError>;
```

Comment:
unclear to me what `release` mean, this is not a keyword that is native to staking. is it unbond, withdraw, or something else? 

name can be something like `unsafe_release_all`

Reply:
Renamed to unsafe_release_all 👍